src/app/Container/D-Observation/ObservationContainer.js

```diff
@@ -0,0 +1,368 @@
+import React from 'react';
+
+// reactstrap components
+import {
+  Card,
+  CardHeader,
+  Table,
+  Row,
+  Badge,
+  CardFooter,
+  DropdownMenu,
+  DropdownItem,
+  UncontrolledDropdown,
+  DropdownToggle,
+  Media,
+  Pagination,
+  PaginationItem,
+  PaginationLink,
+  Button,
+} from 'reactstrap';
+
+class ObservationContainer extends React.Component {
+  render() {
+    return (
+      <>
+        <div className='content'>
+          <div>
+            {' '}
+            <Button
+              block
+              color='primary'
+              //   onClick={() => this.notify('tr')}
+              style={{ width: '20%' }}
+            >
+              Observations
+            </Button>
+          </div>
+          <div>
+            <Row>
+              <div className='col'>
+                <Card className='shadow'>
+                  <CardHeader className='border-0'>
+                    <h3 className='mb-0'>Observations</h3>
+                  </CardHeader>
+                  <Table className='align-items-center table-flush' responsive>
```

Comment:
Can able to create separate component for table and plug with data to it. So we can resue the table component across the application.